core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -243,6 +256,10 @@ public interface TabletsMutator extends AutoCloseable {
    * Interface for changing a tablets persistent data.
    */
   interface TabletMutator {
+    TabletMutator updateLastForAssignmentMode(KeyExtent extent, TServerInstance location);
+
+    TabletMutator updateLastForCompactionMode(Location location, TServerInstance serverInstance);
```

Comment:
I wonder if the same concerns that Keith has for the `updateLastForAssignmentMode` apply here.